daras_ai_v2/billing.py

```
@@ -689,16 +691,25 @@ def _render_seat_selection(plan: PricingPlan, workspace: Workspace) -> SeatSelec
         selected_seat_type = seat_options[seat_type_id]
 
     if plan == PricingPlan.TEAM:
+        default_seat_count = (
+            workspace.subscription
+            and workspace.subscription.billed_seats().count()
```

Ruff (0.15.10)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)